src/main/java/com/enonic/xp/app/welcome/WelcomePageScriptBean.java

```diff
@@ -202,19 +215,101 @@ public Object getSites()
     public Object getProjects()
     {
         List<ProjectJson> projects = new ArrayList<>();
-        projects.addAll( createAdminContext( ContentConstants.CONTENT_REPO_ID ).
-            callWith( () -> projectServiceSupplier.get().list() ).
-            stream().
-            sorted( Comparator.comparing( project -> project.getName().getRepoId().toString() ) ).
-            map( project -> ProjectJson.create().
-                project( project ).
-                iconAsBase64( getProjectIconAsBase64( project ) ).
-                build() ).
-            collect( Collectors.toList() ) );
+        projects.addAll(
+            createAdminContext( ContentConstants.CONTENT_REPO_ID ).callWith( () -> projectServiceSupplier.get().list() ).stream().sorted(
+                Comparator.comparing( project -> project.getName().getRepoId().toString() ) ).map(
+                project -> ProjectJson.create().project( project ).iconAsBase64( getProjectIconAsBase64( project ) ).build() ).collect(
+                Collectors.toList() ) );
 
         return new ProjectsMapper( projects );
     }
 
+    public Object getTemplateApplications()
+    {
+        List<TemplateApplicationJson> templateApps = new ArrayList<>();
+
+        Path filePath = HomeDir.get().toPath().resolve( "config" ).resolve( ".template" );
+        if ( Files.exists( filePath ) )
+        {
+            try
+            {
+                templateApps.addAll( mustParseJsonFile( filePath.toString() ) );
+            }
+            catch ( Exception e )
+            {
+                LOG.error( "Could not parse template file", e );
```

Comment:
```suggestion
                LOG.warn( "Could not parse template file", e );
```
If it is normal that we continue working as is - we don't report an error. warning should be sufficient